tests/test_sample_simple_cur.py

```diff
@@ -1,21 +1,20 @@
 import unittest
 
 import numpy as np
-from sklearn import exceptions
 from sklearn.datasets import fetch_california_housing as load
 
-from skmatter.sample_selection import CUR
+from skmatter.sample_selection import CUR, FPS
 
 
 class TestCUR(unittest.TestCase):
     def setUp(self):
         self.X, _ = load(return_X_y=True)
-        self.X = self.X[:1000]
+        self.X = self.X[FPS(n_to_select=100).fit(self.X).selected_idx_]
         self.n_select = min(20, min(self.X.shape) // 2)
 
     def test_bad_transform(self):
         selector = CUR(n_to_select=2)
-        with self.assertRaises(exceptions.NotFittedError):
+        with self.assertRaises(ValueError):
```

Comment:
Hm... has this been changed somewhere ? I still get NotFittedError

Reply:
inheritance is (ValueError, AttributeError) -> NotFittedError, so I would check the more specific error, because we want to also check the more specific case. You could get a ValueError because of something else and the test passes, but the fitting is not really checked

Reply:
Can you add a patch for what you're suggesting?

Reply:
undoing the change
```suggestion
        with self.assertRaises(exceptions.NotFittedError):
```

Reply:
ah its okay, I tried to change it, then I realized that not fitting is checked here, but really some ValueError. So I just made this more clear by adding doc and renaming the test (because we use the same name for other cases where we are checking if it is fitted).

I'll put my doc changes back in